### ISSUE TITLE
Keyboard in incorrect position with Bootstrap 4 Alpha

### ISSUE THREAD

**Issue by themrflibble** [Mottie/Keyboard#480]
Once upgrading my site from 3.3.4 to 4.0.0-alpha.4, the keyboard now refuses to be positioned.
The console gives the following error 

jquery.min.js:4 Uncaught TypeError: f.getClientRects is not a function

**Comment by Mottie** [comment]
Hi @themrflibble!

I think that is actually an issue with jQuery UI and newer versions of jQuery. If you use jQuery v1.12.0 along with jQuery 3.1.0, it should work.

I tried to set up [a demo](https://jsfiddle.net/Mottie/gfgkb4o1/9/) using Bootstrap 4.0 alpha, but I keep getting an error about it requiring tether. I included it in the resources so I don't know why it's not working.

Either way, I wouldn't recommend using an alpha version in production.
